docs/reference-guide/query-modules/implement-custom-query-modules/custom-query-module-example.md

```diff
@@ -441,6 +585,68 @@ As mentioned before, no exceptions should leave your module. If you are writing
 the module in a language that throws them, use exception handlers
 in `mgp_init_module` and `mgp_shutdown_module` as well.
 
+
+### Batched query procedures
+
+Similar to batched query procedures in Python, you can add batched query procedures in C.
```

Comment:
Do we expect people to reference the Python part or know anything about it? I would kind of consider each language as a separate and individual source of knowledge

Reply:
I think as it is similar, I would not consider it different knowledge.

Reply:
It's not that it's different, just that somebody who is interested in query procedures in C would and first examine how to develop query procedures in Python and have knowledge enough about it that when you say: it's similar to something in the Python API, they say "aha!"

(znaci samo me zanima dal bi netko tko ce ici pisati QM u C-u znati znanje Python APIa da kad se referenciras na Python API oni znati 100% o cemu pricas... ili python developeri citaju samo o pythonu, c dev samo o cu, c++ samo o c++... ili svi citaju sve i zanimaju ih sva 3 APIja?)